### DISCUSSION TITLE
Who needs a 3D printer if there is a Baumarkt (hardware store)

### DISCUSSION THREAD

**Discussion by bmgtr** [jomjol/AI-on-the-edge-device#2471] [Show and tell]
Just used a HT75 pipe piece plus a end cover (75mm Diameter) as a housing. Works like a champ!

![IMG_1620](https://github.com/jomjol/AI-on-the-edge-device/assets/8490086/2b09c53d-be88-4ca7-aa0a-39901f4619b5)

Thanks for this wonderful project!

**Comment by skupu** [comment]
I made it similar way for watermeters of Bmetres - I used 75/50 mm reduction pipe with a 50 mm end cap.
The 75 mm end is taped to the watermeter body and the esp32 cam fits ideally in the 50 mm end cap.
I removed the seal to get easy access to the device.

Power cable goes through the hole in the cap. Antenna is mounted there as well.
Additionally I installed a reset button.

Sorry for not having pictures - I have already lef the installation site, but I will make and paste some when I'll be there again.

The weakpoint is that it has to be taped in order not to be misplaced, as there is no compatibility with the fixing system from the watermeter manufacturer, although it exists.

Here are the pictures of the elements I used:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10194143/b7900787-373c-4c86-8efd-c02c12bc6a53)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10194143/14856b92-4013-4d20-a0bd-14afeb4d2442)
